subsys/bluetooth/mesh/lpn.c

```diff
@@ -556,7 +583,7 @@ int bt_mesh_lpn_friend_offer(struct bt_mesh_net_rx *rx,
 	}
 
 	/* TODO: Add offer acceptance criteria check */
-	k_delayed_work_cancel(&lpn->timer);
+	k_work_cancel_delayable(&lpn->timer);
```

Comment:
can you document here what happens if it fails?

Reply:
Looks like I missed this one. Will do.

Reply:
Thanks.  I've added a recommendation to provide comments like this to the Best Practices section of #33104; I think they really help.

Reply:
:+1: will the best practices be added to the documentation as well?

Reply:
It can be.  Refactoring it to rST is painful and its duplicative of some existing documentation, so I'm going to hold off until it seems complete.

Reply:
I actually ended up deleting this call altogether. It's overridden by a reschedule in the poll below. I think there's a separate issue with the error handling on that, but I will wait for this to go in, and address it separately.